WordPressVIPMinimum/Sniffs/VIP/PHPFilterFunctionsSniff.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\VIP;
+
+use WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * This sniff ensures that proper sanitization is occurring when PHP's filter_* functions are used.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ *
+ * @since 0.4.0
+ */
+class PHPFilterFunctionsSniff extends AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @var string
+	 */
+	protected $group_name = 'php_filter_functions';
+
+	/**
+	 * Functions this sniff is looking for.
+	 *
+	 * @var array The only requirement for this array is that the top level
+	 *            array keys are the names of the functions you're looking for.
+	 *            Other than that, the array can have arbitrary content
+	 *            depending on your needs.
+	 */
+	protected $target_functions = [
+		'filter_var' => true,
+		'filter_input' => true,
+		'filter_var_array' => true,
+		'filter_input_array' => true,
+	];
+
+	/**
+	 * List of restricted filter names.
+	 *
+	 * @var array
+	 */
+	private $restricted_filters = [
+		'FILTER_DEFAULT' => true,
+		'FILTER_UNSAFE_RAW' => true,
+	];
+
+	/**
+	 * Process the parameters of a matched function.
+	 *
+	 * @param int    $stackPtr        The position of the current token in the stack.
+	 * @param array  $group_name      The name of the group which was matched.
+	 * @param string $matched_content The token content (function name) which was matched.
+	 * @param array  $parameters      Array with information about the parameters.
+	 * @return int|void Integer stack pointer to skip forward or void to continue
+	 *                  normal file processing.
+	 */
+	public function process_parameters( $stackPtr, $group_name, $matched_content, $parameters ) {
+		if ( 'filter_input' === $matched_content ) {
+			if ( 2 === count( $parameters ) ) {
```

Comment:
I think I'd like to keep it for standards sake.

Reply:
What "formatting standards"?

Reply:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions

Reply:
> If neither side is a variable, the order is not important.

Makes no difference in terms of avoiding bugs, and whilst there's no variable that could potentially be assigned to, some would find it easier to read "If the count of the parameters is 2, then...", rather than "If 2 is the count of the parameters, then...".